lib/api/v3/utilities/path_helper.rb

```diff
@@ -216,7 +216,7 @@ def self.custom_field_item(id)
           end
 
           def self.custom_field_item_branch(id)
-            "#{custom_field_item(id)}/branch"
+            "#{root}/custom_field_items/#{id}/branch"
```

Comment:
❓ why replacing this? What, if the resource is renamed, then we have here a possible error, that only one of the methods is updated?